Create NOTICE.txt for base microservice (#70)
idrsolutions/base-microservice-example: NOTICE.txt

Base Microservice Example

Copyright 2018 - 2023 IDRsolutions

> This project includes work covered by the following copyright:
> DefaultFileServlet.java - Copyright 2021 OmniFaces

Reviewer: ```suggestion
The DefaultFileServlet class is a derivation of FileServlet by OmniFaces.
Copyright 2021 OmniFaces
```

                        Base Microservice Example
                        =========================

Please visit the IDRSolutions implementations of the base microservice for examples of this project in use:
  * https://github.com/idrsolutions/buildvu-microservice-example
  * https://github.com/idrsolutions/formvu-microservice-example
  * https://github.com/idrsolutions/jpedal-microservice-example

> Copyright 2018 - 2023 IDRsolutions

Reviewer: I'm thinking in this one we should have a mention regarding the DefaultFileServlet class which is derived from OmniFaces.

Author: I have left the OmniFaces copyright in that class.
Shall we also include it here as follows, I took inspiration from other projects.

Copyright 2018 - 2023 IDRsolutions
This project includes work covered by the following copyright:
DefaultFileServlet.java - Copyright 2021 OmniFaces
